tests/integration/components/review-actions-list/review-action/component-collection-test.ts

```diff
@@ -0,0 +1,114 @@
+import { render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+import { setupMirage } from 'ember-cli-mirage/test-support';
+import { setupIntl, TestContext } from 'ember-intl/test-support';
+import { setupRenderingTest } from 'ember-qunit';
+import { module, test } from 'qunit';
+
+import CollectionSubmissionAction, { CollectionSubmissionActionTrigger }
+    from 'ember-osf-web/models/collection-submission-action';
+
+interface ThisTestContext extends TestContext {
+    collectionSubmissionAction: CollectionSubmissionAction;
+}
+
+module('Integration | Component | Collection | Moderator | review-actions', hooks => {
+    setupRenderingTest(hooks);
+    setupIntl(hooks);
+    setupMirage(hooks);
+
+    hooks.beforeEach(async function(this: ThisTestContext) {
+        this.store = this.owner.lookup('service:store');
+        const collection = server.create('collection');
+        const mirageAction = server.create('collection-submission-action', {
+            actionTrigger: CollectionSubmissionActionTrigger.Accept,
+            creator: server.create('user', { fullName: 'Superb Mario' }, 'loggedIn'),
+            target: server.create('collection-submission', {
+                guid: server.create('node', 'withContributors'),
+                collection,
+            }),
+        });
+        this.collectionSubmissionAction = await this.store.findRecord('collection-submission-action', mirageAction.id);
+        this.set('reviewAction', this.collectionSubmissionAction);
```

Comment:
This shouldn't be necessary anymore and we can replace any instances of `this.reviewAction` in this test module with `this. collectionSubmissionAction`